### PR TITLE
cloud: update release notes for 20221115

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
cloud: update release notes for 20221115

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/release-notes-2022.md [hunk 1]
```diff
@@ -8,6 +8,37 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+* The Database Audit Logging feature is now GA.
+
+    You can use Database Audit Logging to record a history of user access details (such as any SQL statements executed) in logs and conduct a periodic analysis of the database audit logs, which helps keep your database secure.
+
+    For detailed information, see [Database Audit Logging](/tidb-cloud/tidb-cloud-auditing.md).
+
+* Support reloading sample data to [Playground](/tidb-cloud/tidb-cloud-quickstart.md#step-2-try-playground).
+
+    In your [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) cluster, if the preloaded sample dataset in **Playground** is deleted, you can now reload the sample data to continue trying **Playground**.
+
```
Comment (TomShawn) [inline tidb-cloud/release-notes-2022.md:41]:
```suggestion
```

File: tidb-cloud/release-notes-2022.md [hunk 2]
```diff
@@ -8,6 +8,33 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+    PITR supports restoring data of any point in time to a new cluster. You can use it to:
+
+    * Reduce RPO in disaster recovery.
+    * Resolve cases of data write errors by restoring point-in-time that is before the error event.
+    * Audit the historical data of the business.
+
+    To use the PITR feature, make sure that your TiDB cluster version is at least v6.3.0 and the TiKV node size is at least 8 vCPU and 16 GiB.
+
+    In addition, with PITR, you can set the backup storage region for clusters located in Tokyo and Osaka regions (for GCP only), so that backup data can be stored in both Tokyo and Osaka, which facilitates data safety and faster recovery. When you restore data, you can choose either region for the new cluster.
```
Comment (TomShawn) [inline tidb-cloud/release-notes-2022.md:23]:
```suggestion
    By default, backup data is stored in the same region where the cluster is created. In Japan, for GCP customers, you can choose to store backup data in one or two regions (Tokyo and/or Osaka). Restoring data from an alternative region provides a higher level of data safety and can accommodate region failures.
```

**Comment by IANTHEREAL** [comment]
LGTM

File: tidb-cloud/release-notes-2022.md [hunk 3]
```diff
@@ -8,6 +8,33 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+    By default, backup data is stored in the same region where the cluster is created. In Japan, for GCP customers with PITR enabled, you can choose to store backup data in one or two regions (Tokyo and/or Osaka). Restoring data from an alternative region provides a higher level of data safety and can accommodate region failures.
+
+    For more information, see [Back up and Restore TiDB Cluster Data](/tidb-cloud/backup-and-restore.md).
+
+    This feature is still in beta and only available upon request:
+
+    * Click **Help** in the lower-right corner of TiDB Cloud console.
+    * In the dialog, fill in "Apply for PITR" in the **Description** field and click **Send**.
+
+* The Database Audit Logging feature is now GA.
```
Comment (ran-huang) [inline tidb-cloud/release-notes-2022.md:32]: The first letters are not capitalized in user doc.
```suggestion
* The database audit logging feature is now GA.
```

File: tidb-cloud/release-notes-2022.md [hunk 4]
```diff
@@ -8,6 +8,33 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+    You can use Database Audit Logging to record a history of user access details (such as any SQL statements executed) in logs and conduct a periodic analysis of the database audit logs, which helps keep your database secure.
```
Comment (ran-huang) [inline tidb-cloud/release-notes-2022.md:34]:
```suggestion
    You can use database audit logging to record a history of user access details (such as any SQL statements executed) in logs and conduct a periodic analysis of the database audit logs, which helps keep your database secure.
```

**Comment by ljun0712** [comment]
LGTM

File: tidb-cloud/release-notes-2022.md [hunk 5]
```diff
@@ -8,6 +8,33 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2022.
 
+## November 15, 2022
+
+* Support point-in-time recovery (PITR) for [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters (beta).
+
+    PITR supports restoring data of any point in time to a new cluster. You can use it to:
+
+    * Reduce RPO in disaster recovery.
+    * Resolve cases of data write errors by restoring point-in-time that is before the error event.
```
Comment (qiancai) [inline tidb-cloud/release-notes-2022.md:18]:
```suggestion
    * Resolve data write errors by restoring point-in-time that is before the error event.
```

**Review by ran-huang — APPROVED** [review]

File: tidb-cloud/release-notes-2022.md [hunk 6]
```diff
@@ -8,6 +8,33 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+    * Audit the historical data of the business.
+
+    To use the PITR feature, make sure that your TiDB cluster version is at least v6.3.0 and the TiKV node size is at least 8 vCPU and 16 GiB.
+
+    By default, backup data is stored in the same region where the cluster is created. In Japan, for GCP customers with PITR enabled, you can choose to store backup data in one or two regions (Tokyo and/or Osaka). Restoring data from an alternative region provides a higher level of data safety and can accommodate region failures.
```
Comment (qiancai) [inline tidb-cloud/release-notes-2022.md:23]:
```suggestion
    By default, backup data is stored in the same region where the cluster is created. In Japan, for TiDB clusters hosted on GCP with PITR enabled, you can choose to store backup data in one or two regions (Tokyo and/or Osaka). Restoring data from an alternative region provides a higher level of data safety and can accommodate region failures.
```

File: tidb-cloud/release-notes-2022.md [hunk 7]
```diff
@@ -8,6 +8,33 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+    This feature is still in beta and only available upon request:
+
+    * Click **Help** in the lower-right corner of TiDB Cloud console.
+    * In the dialog, fill in "Apply for PITR" in the **Description** field and click **Send**.
+
+* The database audit logging feature is now GA.
+
+    You can use database audit logging to record a history of user access details (such as any SQL statements executed) in logs and conduct a periodic analysis of the database audit logs, which helps keep your database secure.
+
+    For detailed information, see [Database Audit Logging](/tidb-cloud/tidb-cloud-auditing.md).
```
Comment (qiancai) [inline tidb-cloud/release-notes-2022.md:36]:
```suggestion
    For more information, see [Database Audit Logging](/tidb-cloud/tidb-cloud-auditing.md).
```

File: tidb-cloud/release-notes-2022.md [hunk 8]
```diff
@@ -8,6 +8,33 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2022.
 
+## November 15, 2022
+
```
Comment (qiancai) [inline tidb-cloud/release-notes-2022.md:12]:
```suggestion

**General changes**

```

**Review by qiancai — APPROVED** [review]
other LGTM

File: tidb-cloud/release-notes-2022.md [hunk 9]
```diff
@@ -8,6 +8,35 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+    PITR supports restoring data of any point in time to a new cluster. You can use it to:
+
+    * Reduce RPO in disaster recovery.
+    * Resolve data write errors by restoring point-in-time that is before the error event.
+    * Audit the historical data of the business.
+
+    To use the PITR feature, make sure that your TiDB cluster version is at least v6.3.0 and the TiKV node size is at least 8 vCPU and 16 GiB.
+
+    By default, backup data is stored in the same region where the cluster is created. In Japan, for TiDB clusters hosted on GCP with PITR enabled, you can choose to store backup data in one or two regions (Tokyo and/or Osaka). Restoring data from an alternative region provides a higher level of data safety and can accommodate region failures.
```
Comment (TomShawn) [inline tidb-cloud/release-notes-2022.md:25]:
```suggestion
    By default, backup data is stored in the same region where the cluster is created. In Japan, for TiDB clusters hosted on GCP with PITR enabled, you can choose to store backup data in one or two regions (Tokyo and/or Osaka). Restoring data from an alternative region provides a higher level of data safety and can tolerate region failures.
```

**Review by lilin90 — APPROVED** [review]
LGTM

**Pull request merged**